src/test/java/com/exasol/adapter/dialects/exasol/ExasolSqlDialectAgainstExasolSevenIT.java

```diff
@@ -0,0 +1,20 @@
+package com.exasol.adapter.dialects.exasol;
+
+import com.exasol.bucketfs.BucketAccessException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.sql.SQLException;
+import java.util.concurrent.TimeoutException;
+
+@Tag("integration")
+@Testcontainers
+public class ExasolSqlDialectAgainstExasolSevenIT extends AbstractExasolSqlDialectIntegrationTest {
+    @BeforeAll
+    static void beforeAll() throws InterruptedException, SQLException, TimeoutException, BucketAccessException {
+        hostAndDefaultPort = "localhost:8563";
```

Comment:
Hardcoding the port is not necessary, since the Container can now tell you the port. That makes the whole duplication unneccessary. A better option for a matrix test is to set the version via system property and have the CI build run the tests for as many versions as required.

Also I want to split the test along the lines of JDBC and EXA.